index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0 shrink-to-fit=no" />
+
+  <title>Library</title>
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body onload="render()">
+  <script src="js/library.js"></script>
+  <main>
+    <div class="container">
+      <h2 class="text-center">Welcome to our library, please add a book below</h2>
+      <div class="book-btn">
+        <button onclick="document.getElementById('id01').style.display='block'" style="width:auto;">Add a book</button>
+      </div>
+      <div id="id01" class="modal">
+        <div class="modal-content animate">
+          <span onclick="document.getElementById('id01').style.display='none'" class="close" title="Close Modal">×</span>
+          <form class="container" id='form'>
+            <div class="form-group">
+              <label for="book_title"><b>Book Title</b></label>
+              <input type="text" placeholder="Book Title" id="book_title" class="form-control" required>
+            </div>
+      
+            <div class="form-group">
+              <label for="book_author"><b>Author</b></label>
+              <input type="text" placeholder="Author" id="book_author" class="form-control" required>
+            </div>
+      
+            <div class="form-group">
+              <label for="pages"><b>No of Pages</b></label>
+              <input type="number" placeholder="Num. Pages" id="pages" class="form-control" required>
+            </div>
+
+            <div class="form-group">
+              <div class="form-check">
+                <input form-check-input type="radio" name="readAlready" value="read" class="form-check-input" id="read">
+                <label class="form-check-label" for="read">Read already</label>
+              </div>
+              <div class="form-check">
+                <input form-check-input type="radio" name="readAlready" value="not read" class="form-check-input" id="not-read">
+                <label class="form-check-label" for="not-read">Not read yet</label>
+              </div>
+            </div>
+
+            <div class="clearfix">
+              <button type="button" onclick="document.getElementById('id01').style.display='none'"
+                class="cancelbtn">Cancel</button>
+              <button onclick="addBookToLibrary()" class="addBook" type="button">Submit</button>
```

Comment:
Calling your `AddBookToLibrary()` function here is causing your `no-unused-var` error.
Ideally you should have an event listener for this button in ur js file.
Please fix 🙏